src/Containers/ClientShowCase.tsx

```diff
@@ -1,34 +1,68 @@
-import React, { Component } from 'react';
+import React from 'react';
 import styled from 'styled-components';
 import { Mixins } from '../Utils';
-import { Button } from '../Inputs/Button';
 
-const IMAGE_HEIGHT = 30;
+const IMAGE_HEIGHTS: object = {
+    small: 15,
+    medium: 30,
+    large: 45,
+    extraLarge: 60,
+};
 
-export interface ClientProps extends React.HTMLAttributes<HTMLDivElement> {
+export interface ShowCaseProps extends React.HTMLAttributes<HTMLDivElement> {
     imgData: string[];
-    handleImageListClick: (
+    handleImageListClick?: (
         event: React.MouseEvent<Element, MouseEvent>,
     ) => void;
-    button: React.ReactNode;
+    button?: React.ReactNode;
```

Comment:
more contextual naming like CenterButton or OnHoverComponent would make it more usable from a readability standpoint